RFC: Rename ANNOUNCE to PUBLISH_NAMESPACE (#1050)
moq-wg/moq-transport: draft-ietf-moq-transport.md

publishers to contact, if any.

An UNSUBSCRIBE_NAMESPACE withdraws a previous SUBSCRIBE_NAMESPACE. It does not
prohibit original publishers from sending further PUBLISH_NAMESPACE or PUBLISH
messages, but relays MUST NOT send any further PUBLISH messages to a client
without knowing the client is interested in and authorized to receive the
content.

## Publishing Namespaces

A publisher MAY send PUBLISH_NAMESPACE messages to any subscriber. A
PUBLISH_NAMESPACE indicates to the subscriber that the publisher has tracks
available in that namespace. A subscriber MAY send SUBSCRIBE or FETCH for tracks
in a namespace without having received a PUBLISH_NAMESPACE for it.

If a publisher is authoritative for a given namespace, or is a relay that has
received an authorized PUBLISH_NAMESPACE for that namespace from an upstream
publisher, it MUST send a PUBLISH_NAMESPACE to any subscriber that has

> subscribed to PUBLISH_NAMESPACE for that namespace, or a more generic set

Reviewer: ```suggestion
subscribed via SUBSCRIBE_NAMESPACE for that namespace, or a more generic set
```
